test/entry_set_test.rb

```diff
@@ -66,9 +66,17 @@ def test_entries
   end
 
   def test_find_entry
-    # by default, #find_entry is case-sensitive
-    assert_equal(ZIP_ENTRIES[0], @zipEntrySet.find_entry('name1'))
-    assert_equal(ZIP_ENTRIES[0], @zipEntrySet.find_entry('NaMe1', false))
```

Comment:
Nope, we do not. Second parameter was introduced by @rastarobot 's implementation and is not needed anymore, superseded by `::Zip.case_insensitive_match`.

Please, obtain a habit of studying the history of the matter before providing your judgement; do not jump to conclusions.

We had a use case where we needed `rubyzip` to behave like DOS-zip (request for "FILE.TXT" should be able to access "file.txt"). @rastarobot implemented it without paying attention to how `rubyzip` accesses files, and added the second parameter to "find_entry", whereas I made a switch that will switch `rubyzip` to *globally* operating like DOS-version.